stand_still_headway_m: 12.0 # Standstill gap between vehicles (m)
max_accel_ms2: 1.5 # Maximum acceleration absolute value used in controller filters (m/s^2)
kp: 0.4 # Proportional weight for PID controller
kd: 0.0  # Derivative Weight for PID controller
ki: 0.03 # Integral weight for PID controller
max_delta_speed_per_timestep: 2.0 # Max value to restrict speed adjustment at one time step (limit on delta_v) (m/s)
min_delta_speed_per_timestep: -10.0 # Min value to restrict speed adjustment at one time step (limit on delta_v) (m/s)
cmd_timestamp_ms: 100 # Timestep to calculate ctrl commands (ms)
adjustment_cap_mps: 15.0 # Adjustment cap for speed command (m/s)
integrator_max: 30.0 # Max limit for integrator term
integrator_min: -30.0  # Min limit for integrator term
vehicle_response_lag: 0.2
enable_max_adjustment_filter: true
enable_max_accel_filter: true